specs/pvs/corla.pvs

```diff
@@ -758,20 +802,23 @@ BEGIN
 
 END roles
 
-canvas_boards: THEORY
+canvass_boards: THEORY
 BEGIN
   IMPORTING roles, election
 
-  canvas_board?: [election -> canvas_board]
-END canvas_boards
+  canvass_board?: [election -> canvass_board]
+END canvass_boards
 
 %+END roles
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-%%% \section{Cryptography}
+%%% \subsection{Cryptography}
 
-%%% We will add additional cryptographic primitives here as required.
+% We will add additional cryptographic primitives here as required.
+% @review kiniry dmz Is it correct to say that the cryptography is used
+% (via hashing) to verify that the ballot manifests and cvrs stored in the 
+% database are actually the ones uploaded by the Counties?
```

Comment:
I'm in alignment with everything that @nealmcb stated here.  Thinking carefully about foolproof means by which 100s of megabyte uploads can happen reliably and securely is important.  Historically I see government agencies purchasing commercial products for such services but they don't work as well as the off-the-shelf open source stuff we use.  E.g., I would think that a secure rsync, or at least a restartable sftp, is wiser than anything else in this regards.